docs/remote_inference_blueprints/gcp_vertexai_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,267 @@
+### GCP VertexAI Embedding Connector Blueprint:
+
+## 1. Add VertextAI endpoint to trusted URLs:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://.*-aiplatform\\.googleapis\\.com/.*$"
+        ]
+    }
+}
+```
+
+## 2. Generate access-token to access Vertex AI
+
+Vertex AI API can be only accessed by using the access tokens, you can generate the short-live or long live (upto 12h)
+
+short live token can be generated using below command:
+
+```
+gcloud auth print-access-token
+```
+
+Longer expriration token upto 12h, can be generated using the below command:
```

Comment:
expiration

Reply:
@mingshl As the google cloud services, specially VertexAI  is only accessible using the token, these token is generated by the command "gcloud auth print-access-token", the token generated has some expiration default value is 1-2 hours but if it generated using the service account, it can have expiration upto 12h.